src/components/atoms/Button/Button.tsx

```diff
@@ -44,6 +46,8 @@ export const AppButton: React.FC<ButtonProps> = ({
         className={customClass}
         style={customStyle}
         to={linkTo!}
+        target={newTab ? "_blank" : undefined}
+        rel={newTab ? "noopener noreferer" : undefined}
```

Comment:
```suggestion
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1705/commits/f038fa7f07dd402677ba46b787847df727270978